package.json

```
@@ -47,7 +47,7 @@
     "mdast-util-mdxjs-esm": "^2.0.0",
     "property-information": "^6.0.0",
     "space-separated-tokens": "^2.0.0",
-    "style-to-object": "^1.0.0",
+    "style-to-object": "^1.0.5",
```

I'd defer to @wooorm, generally `syntax-tree` prefers wider version ranges to allow deduplication.
```suggestion
    "style-to-object": "^1.0.0",
```

---

Makes sense! Updated b8da427